nautobot/core/api/schema.py

```diff
@@ -36,6 +37,49 @@ class NautobotAutoSchema(AutoSchema):
         }
     )
 
+    @property
+    def is_bulk_action(self):
+        """Custom property for convenience."""
+        return hasattr(self.view, "action") and self.view.action in self.custom_actions
+
+    @property
+    def is_partial_action(self):
+        """Custom property for convenience."""
+        return hasattr(self.view, "action") and self.view.action in ["partial_update", "bulk_partial_update"]
+
+    def _get_paginator(self):
+        """Nautobot's custom bulk operations, even though they return a list of records, are NOT paginated."""
+        if self.is_bulk_action:
+            return None
+        return super()._get_paginator()
+
+    def get_filter_backends(self):
+        """Nautobot's custom bulk operations, even though they return a list of records, are NOT filterable."""
+        if self.is_bulk_action:
+            return []
+        return super().get_filter_backends()
```

Comment:
Regarding v2 - I don't think it makes sense for bulk-PATCH and bulk-PUT to support filter parameters or to provide true paginated responses (how would you retrieve "page 2" of a PATCH response?). I'd like to push an issue up to drf-spectacular to make `get_paginator` into a non-private method though.